listenbrainz/webserver/static/js/src/PlaybackControls.tsx

```diff
@@ -0,0 +1,154 @@
+import * as React from "react";
+import {
+  faEye,
+  faEyeSlash,
+  faFastBackward,
+  faFastForward,
+  faPauseCircle,
+  faPlayCircle,
+  faSortAmountDown,
+  faSortAmountUp,
+} from "@fortawesome/free-solid-svg-icons";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { IconDefinition } from "@fortawesome/fontawesome-common-types"; // eslint-disable-line import/no-unresolved
+import { IconProp } from "@fortawesome/fontawesome-svg-core";
+
+export type SpotifyPlayDirection = "up" | "down" | "hidden";
```

Comment:
I think we should keep all the types in the types.d.ts file, that ensures re usability

Reply:
Moved the non prop and non state types to types.d.ts. The propstype and statetype are gonna stay here, as agreed on irc.